modules/settings/variables.tf

```
@@ -53,34 +53,22 @@ variable "custom_image_tag" {
   EOD
 }
 
-variable "installation_type" {
-  type        = string
-  description = "(Required) Installation type for Terraform Enterprise"
-
-  validation {
-    condition = (
-      var.installation_type == "poc" ||
-      var.installation_type == "production"
-    )
-    error_message = "The installation type must be 'production' (recommended) or 'poc' (only used for demo-mode)."
-  }
-}
-
 variable "production_type" {
-  default     = null
+  default     = "disk"
```

We want this module to be completely unopinionated, including with defaults, so this would remain `null`.
```suggestion
  default     = null
```